homeassistant/components/tado/const.py

```diff
@@ -48,6 +48,12 @@
 DATA = "data"
 UPDATE_TRACK = "update_track"
 
+CONFIGURATION_URL_HOME = "https://app.tado.com"
```

Comment:
If constants are only used in one module, it is recommented to keep them in that module.

Reply:
Really?
I don't agree with that...

Reply:
> If I remember correctly the general idea was to avoid creating platform specific constants in const.py as it made it harder to review.

I personally would prefer to declare all constants for a module in a single place. But... opinion 🤷‍♂️

If anything, I don't think the constants are needed in this PR at all. None of these cases are used in multiple places. IMHO this should just be added directly to the descriptions and remove the constants altogether.

@avee87 Please consider removing the constants instead.